post-processor/googlecompute-import/post-processor.go

```diff
@@ -36,6 +36,17 @@ type Config struct {
 	AccountFile string `mapstructure:"account_file" required:"false"`
 	// This allows service account impersonation as per the [docs](https://cloud.google.com/iam/docs/impersonating-service-accounts).
 	ImpersonateServiceAccount string `mapstructure:"impersonate_service_account" required:"false"`
+	// The service account scopes for launched
+	// instance. Defaults to:
+	//
+	// ```json
+	// [
+	//   "https://www.googleapis.com/auth/userinfo.email",
+	//   "https://www.googleapis.com/auth/compute",
+	//   "https://www.googleapis.com/auth/devstorage.full_control"
```

Comment:
Hello there thanks for updating this and no worries . :)

Another question here, where is this default happening here ? In the plugin codebase or over on GCP side ?

- If it is the first case, then I haven't found the code doing that.
- If it is the latter, then I think that we should point at the remote docs here.

Reply:
Ok, I got it now. I finally committed sin out of ignorance.
I was totally unaware of the other options that the plugin offers (particularly the post-process `import` and `export`). My comments in the code are indeed wrong.
I will make additional tests and check that these use-cases are well covered, and finally will update the code/doc accordingly.